irida_import/tests/unit/test_irida_import.py

```diff
@@ -129,7 +129,7 @@ def test_get_samples(self, imp, setup_json):
         """
         param_dict = json.loads(setup_json)['param_dict']
         json_params = json.loads(param_dict['json_params'])
-        sample_file = SampleFile('nameish', 'pathish')
+        sample_file = SampleFile('nameish', 'pathish', "http://127.0.0.1/api/samples/1/pairs/1/files/1")
```

Comment:
It's also good to state the arguments in test cases, because if something changes you can spot the bug instantly, instead of having to go searching for where the problem is.
```
sample_file = SampleFile(
    name='nameish',
    path='pathish',
    href="http://127.0.0.1/api/samples/1/pairs/1/files/1"
)
```

Reply:
Updated in 97c187bc8ab0716b21f78c299f47c20ad4f1ae00